students/kegan/session03/mailroom.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python3
+
+"""
+Kathryn Egan
+
+This program manages donors and their donations for
+Miuvenile Care, a charity organizaiton for disadvantaged
+kittens.
+"""
+
+
+DONORS = {
+    'Benedict Cumberbatch': [200000.0],
+    'Elon Musk': [10000.0, 150000.0, 100000.0],
+    'Dad': [20.0, 5.0],
+    'Donald Trump': [2.81],
+    'Billy Neighbor': [.54, .01, .25]}
+
+COLUMN_WIDTHS = [0] * 4
```

Comment:
I wonder if it's worth having this as a global, rather than simply re-creating it when you need it.